<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>897</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>891</width>
      <height>561</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-image: url(:/img/bg2.jpg);</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>326</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="collectionButton">
       <property name="text">
        <string>Multimedia collections</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BorrowerButton">
       <property name="text">
        <string>Borrowers</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="itemButton">
       <property name="text">
        <string>Items</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>251</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QPushButton" name="submitItem">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>180</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>226</width>
       <height>170</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>idenCode</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Bar Code</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_2"/>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>Borrow Status</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>Borrow Times</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit_4"/>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="pushButton_5">
        <property name="text">
         <string>Date Borrow</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="pushButton_6">
        <property name="text">
         <string>Numbers of Day</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="lineEdit_6"/>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBox3">
        <item>
         <property name="text">
          <string>In Library</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>In Store</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Lost</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Borrowed</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Damaged</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDateEdit" name="dateEdit">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_borrow" native="true">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>70</y>
      <width>261</width>
      <height>231</height>
     </rect>
    </property>
    <widget class="QPushButton" name="AddBorrower">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>190</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>220</width>
       <height>186</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton_7">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_10"/>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButton_8">
        <property name="text">
         <string>Address</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_11"/>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pushButton_9">
        <property name="text">
         <string>Department</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>IT</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>BCOMM</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Design</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="pushButton_10">
        <property name="text">
         <string>Mobile</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit_12"/>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="pushButton_11">
        <property name="text">
         <string>Item Borrowed</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEdit_13"/>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="pushButton_12">
        <property name="text">
         <string>Item Late</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="lineEdit_14"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_collection" native="true">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>80</y>
      <width>251</width>
      <height>311</height>
     </rect>
    </property>
    <widget class="QPushButton" name="Add_Collection">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>270</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>243</width>
       <height>257</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton_13">
        <property name="text">
         <string>Type</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox_2">
        <item>
         <property name="text">
          <string>CD</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DVD</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButton_14">
        <property name="text">
         <string>Data Type</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pushButton_15">
        <property name="text">
         <string>Title</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_20"/>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="pushButton_16">
        <property name="text">
         <string>Version</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit_21"/>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="pushButton_17">
        <property name="text">
         <string>Iden Code</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEdit_22"/>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="pushButton_18">
        <property name="text">
         <string>Publisher</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="lineEdit_23"/>
      </item>
      <item row="6" column="0">
       <widget class="QPushButton" name="pushButton_19">
        <property name="text">
         <string>Year Recored</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="lineEdit_24"/>
      </item>
      <item row="7" column="0">
       <widget class="QPushButton" name="pushButton_20">
        <property name="text">
         <string>Duration</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="lineEdit_25"/>
      </item>
      <item row="8" column="0">
       <widget class="QPushButton" name="pushButton_21">
        <property name="text">
         <string>Category</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBox_3">
        <item>
         <property name="text">
          <string>Image</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Audio</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Video</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QComboBox" name="comboBox_4">
        <item>
         <property name="text">
          <string>Engineering and Technology</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>General</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Business and Economics</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>History</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Film and media studies</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Psychology</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Images</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>photographs&amp; maps</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_22">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Display</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_23">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Function</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>350</y>
      <width>321</width>
      <height>281</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>280</y>
      <width>291</width>
      <height>261</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>graphicsView</zorder>
   <zorder>widget</zorder>
   <zorder>widget_borrow</zorder>
   <zorder>widget_collection</zorder>
   <zorder>pushButton_22</zorder>
   <zorder>pushButton_23</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>layoutWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>897</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuOption">
    <property name="title">
     <string>Option</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOption"/>
   <addaction name="menuAbout"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
